be2-scala/linter/project/plugins.sbt

```diff
@@ -1,3 +1,4 @@
 resolvers += Resolver.sonatypeRepo("releases")
-addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.12.0")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.12.1")
 addSbtPlugin("com.eed3si9n" % "sbt-projectmatrix" % "0.9.2")
+addSbtPlugin("com.geirsson" % "sbt-ci-release" % "1.5.7")
```

Comment:
Why do we need that ?

Reply:
It's to be able to publish the rules to Maven using sbt publish